pegasus/sites.v3/hourofcode.com/views/events_signup.haml

```diff
@@ -0,0 +1,9 @@
+%a#hocsignupform{name: 'signup'}
+- if ["actual-hoc", "post-hoc", false].include?(hoc_mode)
```

Comment:
Is this `false` here intentionally?

Reply:
I think it's there just in case `hoc-mode` is ever set to `false`, which is technically possible but unlikely. If I remember correctly it'd act the same as `post-hoc` mode.

Reply:
Gotcha, thank you for clarifying!